Tests/Functional/FunctionalTest.php

```diff
@@ -49,25 +50,41 @@
     /**
      * @return void
      */
-    public function setUp() {
+    public function setUp()
+    {
         parent::setUp();
         $this->objectManager = GeneralUtility::makeInstance('TYPO3\\CMS\\Extbase\\Object\\ObjectManager');
     }
 
     /**
+     * Loads a Fixture from the Fixtures folder beside the current test case.
+     *
      * @param $fixtureName
      * @throws \TYPO3\CMS\Core\Tests\Exception
      */
-    protected function importDataSetFromFixture($fixtureName) {
-        $this->importDataSet(__DIR__ . '/Fixtures/' . $fixtureName);
+    protected function importDataSetFromFixture($fixtureName)
+    {
+        $this->importDataSet($this->getRuntimeDirectory() . '/Fixtures/' . $fixtureName);
+    }
+
+    /**
+     * Returns the directory on runtime.
+     *
+     * @return string
+     */
+    protected function getRuntimeDirectory()
+    {
+        $rc = new \ReflectionClass(get_class($this));
+        return dirname($rc->getFileName());
```

Comment:
Hi Ingo, this is because you want to place fixtures by convention beside every testcase in the "Fixtures" folder __DIR__ would only be the folder beside the AbstractTestCase and i think it is better to place the fixtures right beside the TestCase. But in general i would agree to avoid using reflection whenever possible.